src/aat/resources/features/F-051 - Get Profile/S-109.td.json

```diff
@@ -329,6 +329,8 @@
 								}
 							],
 							"searchAliasFields": [],
+              "searchParties" : [],
```

Comment:
Whilst I know the data-store needs to consume these GlobalSearch config fields but does it need to supply this in its own API calls that contain case type information?    i.e. should / could we be hiding it?

Reply:
Could the same be said of `searchAliasFields` and other config fields returned with no values ?

If we need to hide - I guess this is a BA/tech lead decision ?

Reply:
i guess it's coming from common Model object. If so..leave as it is for now (until we have a separate DTOs for each endpoint.)